app/views/organisations/_expanded_works_with_organisations.html.erb

```diff
@@ -0,0 +1,17 @@
+<a href="#" data-controls="toggle_<%= current_organisation.parameterize %>" data-expanded="false" data-toggled-text="<%= t('view_less') %>">
```

Comment:
Good catch

Reply:
I'd have gone with just putting it onto the next line by default for mobile, in order to avoid an inconsistent layout, as shown:

<img width="337" alt="screen shot 2018-06-11 at 11 45 24" src="https://user-images.githubusercontent.com/861310/41227524-0c442908-6d6d-11e8-9aa1-5d94bbca23ab.png">

Reply:
@andysellick Good point, I'll change that